app/src/main/java/org/hackillinois/android/view/maps/MapsFragment.kt

```diff
@@ -0,0 +1,90 @@
+package org.hackillinois.android.view.maps
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.FrameLayout
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_maps.view.*
+import org.hackillinois.android.R
+
+class MapsFragment : Fragment() {
+
+    private lateinit var outdoorMapsButton: Button
+    private lateinit var indoorMapsButton: Button
+
+    private var outdoors = true
+
+    private lateinit var contentFrame: FrameLayout
+
+    private val outdoorMapsFragment = OutdoorMapsFragment.newInstance()
+    private val indoorMapsFragment = IndoorMapsFragment.newInstance()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_maps, container, false)
+
+        outdoorMapsButton = view.findViewById(R.id.outdoorButton)
+        indoorMapsButton = view.findViewById(R.id.indoorButton)
+        contentFrame = view.findViewById(R.id.mapsContainer)
+
+        outdoorMapsButton.setOnClickListener(outdoorMapsClickListener)
+        indoorMapsButton.setOnClickListener(indoorMapsClickListener)
+        view.tabLayout.addOnTabSelectedListener(tabListener)
+
+        outdoorMapsButton.isSelected = true
```

Comment:
I think we should switch these two. Indoor maps should be to the left of outdoor maps, and indoor maps should start as selected.

Reply:
Reasoning is that indoor maps load faster than outdoor maps